docs/api-docs/catalog/v2-v3-examples.md

```diff
@@ -0,0 +1,1052 @@
+    "tax_class_id": 0,
+    "option_set_display": "right",
+    "bin_picking_number": "",
+    "custom_url": "/fog-linen-chambray-towel-beige-stripe/",
+    "primary_image": {
+        "id": 266,
+        "zoom_url": "https://cdn11.bigcommerce.com/{store_hash}/products/77/images/266/foglinenbeigestripetowel1b.1580767715.1280.1280.jpg?c=1",
+        "thumbnail_url": "https://cdn11.bigcommerce.com/{store_hash}/products/77/images/266/foglinenbeigestripetowel1b.1580767715.220.290.jpg?c=1",
+        "standard_url": "https://cdn11.bigcommerce.com/{store_hash}/products/77/images/266/foglinenbeigestripetowel1b.1580767715.386.513.jpg?c=1",
+        "tiny_url": "https://cdn11.bigcommerce.com/{store_hash}/products/77/images/266/foglinenbeigestripetowel1b.1580767715.44.58.jpg?c=1"
+    },
+    "availability": "available",
+    "brand": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/brands/0",
+        "resource": "/brands/0"
+    },
+    "downloads": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/downloads",
+        "resource": "/products/77/downloads"
+    },
+    "images": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/images",
+        "resource": "/products/77/images"
+    },
+    "videos": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/videos",
+        "resource": "/products/77/videos"
+    },
+    "skus": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/skus",
+        "resource": "/products/77/skus"
+    },
+    "rules": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/rules",
+        "resource": "/products/77/rules"
+    },
+    "option_set": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/optionsets/14",
+        "resource": "/optionsets/14"
+    },
+    "options": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/options",
+        "resource": "/products/77/options"
+    },
+    ...
+    },
+    "metadata": []
+}
+```
+
+### Create a product
+
+To create a simple product using the V3 Catalog API, send a `POST` request to `/v3/catalog/products`.
+
+**V3 request**
+
+`POST /v3/catalog/products`
+
+```json
+{
+  "name": "BigCommerce Storage Basket",
+  "price": "10.00",
+  "categories": [
+    23,
+    21
+  ],
+  "weight": 4,
+  "type": "physical"
+}
+```
+
+To create a simple product using the V2 Products API, in addition to the properties in the V3 example, your `POST` request must include `availability`.
+
+**V2 request**
+
+`POST /v2/products`
+
+```json
+{
+  "name": "BigCommerce Clay Vase",
+  "price": "20.00",
+  "categories": [
+    23,
+    21
+  ],
+  "weight": 5,
+  "type": "physical",
+  "availability": "available"
+}
+```
+
+### Create a product with images
+
+To create a simple product with an image using the V3 Catalog API, first create the product, then add the image.
```

Comment:
Usually, you'd have a comma after "first" since it is part of a series. Since you've already outlined how to create a product above, maybe this section could be titled "Adding a product image" and specify that the product must already exist. Then there isn't a need for a series at all.